### PR TITLE
feat(#614) Increase application and method priority. 

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
Priority of `htail` components is changed a little bit in order to handle `methods` correctly. In other words, "dot" priority now become higher. 
Please, see the test for more info.

Closes: #614

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@mximp please, check

**Review by mximp — APPROVED** [review]

**Comment by volodya-lombrozo** [comment]
@yegor256 please, merge

File: eo-parser/src/test/java/org/eolang/parser/PriorityTest.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            MatcherAssert.assertThat(
+                new XMLDocument(baos.toByteArray())
+                    .nodes("//o[@base='.plus']")
+                    .get(0).nodes(String.format("//o[@base='%s']", base)),
+                Matchers.not(Matchers.empty())
+            );
+        }
+    }
+
+    static Stream<Arguments> plusTestCases() {
```
Comment (yegor256) [inline eo-parser/src/test/java/org/eolang/parser/PriorityTest.java:91]: @volodya-lombrozo why can't you use PacksTest for this? what's the point of creating a new JUnit test?

Comment (volodya-lombrozo) [reply eo-parser/src/test/java/org/eolang/parser/PriorityTest.java]: @yegor256 You absolutely right, we don't need it. I've moved the logic to `PacksTest`. Please, check.

**Comment by volodya-lombrozo** [comment]
@yegor256 merge, please

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**